src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.spec.ts

```diff
@@ -251,6 +251,24 @@ describe('ServiceFormComponent', () => {
         const ssl_key = fixture.debugElement.query(By.css('#ssl_key'));
         expect(ssl_key).toBeNull();
       });
+
+      it('should test .pem file', () => {
+        let pemCert = `-----BEGIN CERTIFICATE-----
+        iJ5IbgzlKPssdYwuAEI3yPZxX/g5vKBrgcyD3LttLL/DlElq/1xCnwVrv7WROSNu
+        -----END CERTIFICATE-----
+        -----BEGIN CERTIFICATE-----
+        mn/S7BNBEC7AGe5ajmN+8hBTGdACUXe8rwMNrtTy/MwBZ0VpJsAAjJh+aptZh5yB
+        -----END CERTIFICATE-----
+        -----BEGIN RSA PRIVATE KEY-----
+        x4Ea7kGVgx9kWh5XjWz9wjZvY49UKIT5ppIAWPMbLl3UpfckiuNhTA==
+        -----END RSA PRIVATE KEY-----`;
```

Comment:
Minor thing: when using multiline literals (known as heredoc in scripting world) it's usually accepted to break the indentation (rather than fixing that later):
```suggestion
        let pemCert =`
-----BEGIN CERTIFICATE-----
iJ5IbgzlKPssdYwuAEI3yPZxX/g5vKBrgcyD3LttLL/DlElq/1xCnwVrv7WROSNu
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
mn/S7BNBEC7AGe5ajmN+8hBTGdACUXe8rwMNrtTy/MwBZ0VpJsAAjJh+aptZh5yB
-----END CERTIFICATE-----
-----BEGIN RSA PRIVATE KEY-----
x4Ea7kGVgx9kWh5XjWz9wjZvY49UKIT5ppIAWPMbLl3UpfckiuNhTA==
-----END RSA PRIVATE KEY-----`;
```